File: team3/multiplication.py
import numbers

class Multiplication:
    def multiply(self, a, b):
        # numbers.Number: int, float 등 숫자 타입을 포괄
        if not isinstance(a, numbers.Number) or not isinstance(b, numbers.Number):
            return "숫자만 입력하세요."
        
        result = a * b
        
        # float 결과에 대해 소수점 오차 제거 (소수점 10자리까지 반올림)
        if isinstance(result, float):
            return round(result, 10)
        return result

File: calculator/src/Multiplication.java
//곱셈 기능 담당 클래스
public class Multiplication {
    public static double multiply(double num1, double num2) {    //실수형 매개변수 2개 받는 매서드 static으로 선언
        return num1 * num2;                                //  num1과 num2를 곱한 값 반환
    }
}

/*사용 예시
public class Main {
    public static void main(String[] args) {
        double result = Multiplication.multiply(4, 5);      // 두 수의 곱셈 결과 result변수에 저장
        System.out.print("result = " + result);             // 결과 출력
    }
}
*/